web/src/slices/credentials/credentialsSlice.ts

```diff
@@ -47,8 +58,7 @@ const credentialSlice = createSlice({
         const results = action.payload.results
         let revocationObjects: RevocationRecord[] = []
         if (results?.length) {
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          results.forEach((cred: any) => {
+          results.forEach((cred: CredentialItem) => {
```

Comment:
Preferred to apply the type definition to `results` (root of the problem) than to the array method's callback's params.

Reply:
Completed